reconstruction/alert/src/main/java/org/jlab/rec/ahdc/Distance/CircleFitter.java

```diff
@@ -0,0 +1,365 @@
+package org.jlab.rec.ahdc.Distance;
+
+import Jama.Matrix;
```

Comment:
We are trying to drop the dependency on Jama: if possible can you switch to a different matrix library ?

Reply:
addressed by switching to org.apache.commons.math3